code/go/0chain.net/chaincore/block/entity.go

```diff
@@ -589,13 +589,17 @@ func (b *Block) GetClients() ([]*client.Client, error) {
 		if _, ok := cmap[t.PublicKey]; ok {
 			continue
 		}
-		c, err := client.GetClientFromCache(t.ClientID)
```

Comment:
We'd better not remove the clients cache as `c.SetPublicKey` is a cpu intensive process, we need to avoid it as much as possible.